src/Twig/MediaRuntime.php

```diff
@@ -150,7 +152,12 @@ private function getMedia($media): ?MediaInterface
             return null;
         }
 
-        if (MediaInterface::STATUS_OK !== $media->getProviderStatus()) {
+        $fileName = $media->getName();
+
+        if (
+            MediaInterface::STATUS_OK !== $media->getProviderStatus() &&
+            (null !== $fileName && \in_array(strtolower(pathinfo($fileName, \PATHINFO_EXTENSION)), self::EXCLUDED_FORMATS, true))
+        ) {
```

Comment:
You added a check in `LiipImageThumbmail` about svg. Why you need one here too ?

Also, it's supposed to be
```
MediaInterface::STATUS_OK !== $media->getProviderStatus()
|| YourExtraCheck
```

Reply:
Ok i'll check this

Reply:
a check in the file is needed due to the fact that the uploaded svg image is assigned a MediaInterface::STATUS_ERROR and if this check is not done, null will always be returned

Reply:
Maybe we should also solve the STATUS_ERROR then? It should be because of MediaBundle trying to create thumbnails for an SVG when uploading it.

Reply:
Indeed, might be better to handle the case status_error and svg extension this way.

Reply:
Ok, i'll try to solve status_error in the next week